tests/test_filtsmooth/test_gaussfiltsmooth/test_kalmanposterior.py

```diff
@@ -81,6 +81,7 @@ def test_call_extrapolation(self):
 
 
 class TestKalmanPosteriorSampling(CarTrackingDDTestCase, NumpyAssertions):
+    # There is no check as to whether the samples make sense...
```

Comment:
Should we open an issue for this? If not, feel free to just resolve

Reply:
I will